src/test/java/de/mediathekview/mserver/crawler/CrawlerManagerTest.java

```diff
@@ -45,7 +42,7 @@ public CrawlerManagerTest(final String aFilmlistPath, final FilmlistFormats aFor
   @Parameterized.Parameters(name = "Test {index} Filmlist for {0} with {1}")
   public static Collection<Object[]> data() {
     return Arrays
-        .asList(new Object[][] {{"filmlists/TestFilmlistNewJson.json", FilmlistFormats.JSON},
+        .asList(new Object[][]{{"filmlists/TestFilmlistNewJson.json", FilmlistFormats.JSON},
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)